.github/workflows/markdownlint/config.yaml

```diff
@@ -0,0 +1,12 @@
+# All rules are inactive by default.
+default: false
+
+# These specific rules are active.
+# See https://github.com/DavidAnson/markdownlint#rules--aliases for details.
+heading-increment: true
+no-reversed-links: true
+no-missing-space-atx: true
+no-multiple-space-atx: true
+blanks-around-lists: true
+no-alt-text: true
```

Comment:
I checked our current content and suggest to remove those last two lines:
* no-alt-text: I found dozens of examples in our own Markdown output where there is no alternative text given
* blanks-around-lists: Before we switch this on, I'd like to run a lint on our own content. Not sure if we comply to this...

The other ones look ok to me.

```suggestion
```